### PR TITLE
Update rjsf theme readmes

### LINKED ISSUE #1728: Add a README for @rjsf/core
### Description

The npm link for @rjsf/core has an empty readme: https://www.npmjs.com/package/@rjsf/core

We should add a file `packages/core/README.md`. We can model it off of the excellent README already there for @rjsf/material-ui: https://www.npmjs.com/package/@rjsf/material-ui

### PR DESCRIPTION
**Pull request by epicfaace**
Make all READMEs for the themes consistent and look nice -- it's all based on the original material-ui README, thanks @agustin107 for the nice template!

Also fixes #1728.

Sample README (for top-level directory):

image

### CODE REVIEW

**Comment by epicfaace** [comment]
@stanlemon I've changed this enough since your review that I'm going to ask for another review from you. Thanks!

**Comment by epicfaace** [comment]
This was a lot of copying-and-pasting, but it's probably worth it because it will make the npm package pages look better. I'm thinking we could probably make a README template with the common parts of each README in the future and introduce a build step to actually generate READMEs for npm, though.

File: packages/antd/README.md [hunk 1]
```diff
@@ -1,40 +1,63 @@
...
 <p align="center">
+  <a href="https://github.com/rjsf-team/react-jsonschema-form">
+    <img src="https://raw.githubusercontent.com/rjsf-team/react-jsonschema-form/59a8206e148474bea854bbb004f624143fbcbac8/packages/antd/logo.png" alt="Logo" width="120" height="120">
+  </a>
+
   <h3 align="center">@rjsf/antd</h3>
 
   <p align="center">
-    Ant Design theme, fields and widgets for <a href="https://github.com/rjsf-team/react-jsonschema-form"><code>react-jsonschema-form</code></a>.
+  Ant Design theme, fields and widgets for <a href="https://github.com/mozilla-services/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
```
Comment (stanlemon) [inline packages/antd/README.md:16]: This url looks wrong, no?

File: packages/antd/README.md [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+Ant Design theme, fields and widgets for `react-jsonschema-form`.
+
+### Built With
+
+- [react-jsonschema-form](https://github.com/mozilla-services/react-jsonschema-form/)
```
Comment (stanlemon) [inline packages/antd/README.md:50]: This url too

File: packages/core/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+Core logic and classic Bootstrap 3 theme for `react-jsonschema-form`.
+
+### Built With
+
+- [react-jsonschema-form](https://github.com/mozilla-services/react-jsonschema-form/)
```
Comment (stanlemon) [inline packages/core/README.md:50]: This URL

File: packages/material-ui/README.md [hunk 4]
```diff
@@ -1,39 +1,58 @@
...
-    <img src="https://raw.githubusercontent.com/cybertec-postgresql/rjsf-material-ui/master/rjsf-material-ui-logo.png" alt="Logo" width="140" height="120">
+  <a href="https://github.com/rjsf-team/react-jsonschema-form">
+    <img src="https://raw.githubusercontent.com/rjsf-team/react-jsonschema-form/59a8206e148474bea854bbb004f624143fbcbac8/packages/material-ui/logo.png" alt="Logo" width="120" height="120">
   </a>
 
   <h3 align="center">@rjsf/material-ui</h3>
 
   <p align="center">
-  Material-UI theme, fields and widgets for <a href="https://github.com/mozilla-services/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
+  Material UI theme, fields and widgets for <a href="https://github.com/mozilla-services/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
```
Comment (stanlemon) [inline packages/material-ui/README.md:17]: This URL

File: packages/semantic-ui/README.md [hunk 5]
```diff
@@ -1,58 +1,41 @@
...
-    <img src="https://react.semantic-ui.com/logo.png" alt="Logo" width="140" height="120">
+  <a href="https://github.com/rjsf-team/react-jsonschema-form">
+    <img src="https://raw.githubusercontent.com/rjsf-team/react-jsonschema-form/59a8206e148474bea854bbb004f624143fbcbac8/packages/semantic-ui/logo.png" alt="Logo" width="120" height="120">
   </a>
 
   <h3 align="center">rjsf-semantic-ui</h3>
 
   <p align="center">
-  Semantic-UI theme, fields and widgets for <a href="https://github.com/mozilla-services/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
+  Semantic UI theme, fields and widgets for <a href="https://github.com/mozilla-services/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
```
Comment (stanlemon) [inline packages/semantic-ui/README.md:17]: This URL

**Pull request merged**